### PR TITLE
Revision of Naming: replacing "Delta" with "Epsilon"

### PR DESCRIPTION
**Pull request by louisj381**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,4 @@
-SchulichDeltaDesktopTelemetry.pro.*
+SchulichEpsilonDesktopTelemetry.pro.*
```
Comment (ExtraGravity) [inline .gitignore]: This should just be `*.pro.*`

File: SchulichEpsilonHermes/SchulichEpsilonHermes.cpp [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#include "InfrastructureLayer/InfrastructureContainer.h"
```
Comment (ExtraGravity) [inline SchulichEpsilonHermes/SchulichEpsilonHermes.cpp]: I don't see the old files removed in this PR, they should be removed in this PR.

**Review by ExtraGravity — CHANGES_REQUESTED** [review]

**Comment by ExtraGravity** [comment]
Your build is also failing, get that patched up.

File: BusinessLayer/LoggerService/LoggerService.cpp.bak [hunk 3]
```diff
@@ -0,0 +1,174 @@
+#include <QDebug>
```
Comment (bill-luu) [inline BusinessLayer/LoggerService/LoggerService.cpp.bak]: Please remove this file. `.bak` files come from running AStyle and are the original files before being edited. Usually these are removed in the `format.sh`, so I'm not really sure how this is here O_o

**Review by bill-luu — CHANGES_REQUESTED** [review]

File: BusinessLayer/LoggerService/LoggerService.cpp.bak (hunk 3 above)
Comment (bill-luu) [reply BusinessLayer/LoggerService/LoggerService.cpp.bak]: Actually, `.orig` files are the ones that come from AStyle....this `.bak` file must have been from somewhere else then

**Comment by bill-luu** [comment]
There is a Merge Conflict too, so please fix that.

**Comment by bill-luu** [comment]
The `.bak` file still needs to be removed.

**Pull request merged**